kernel_gateway/notebook_http/__init__.py

```diff
@@ -63,14 +63,17 @@ def __init__(self, *args, **kwargs):
         # Build the parser using the comment syntax for the notebook language
         func = getattr(cell_parser_module, 'create_parser')
         try:
-            lang = self.parent.seed_source['metadata']['language_info']['name']
+            kernel_language = self.parent.seed_notebook['metadata']['language_info']['name']
         except (AttributeError, KeyError):
-            lang = None
-        prefix = self.comment_prefix[lang]
+            kernel_language = None
+        if kernel_language in self.comment_prefix:
```

Comment:
Instead of if/else, please use the pythonic lookup:

    kernel_language = self.comment_prefix.get(kernel_language, '#')

Then you can also remove the value for `None` in the comment_prefix dict.

Reply:
ok~